src/main/java/io/stargate/sgv2/jsonapi/service/embedding/DataVectorizer.java

```diff
@@ -263,19 +264,17 @@ public <T> Uni<T> vectorizeTasks(
     // so quick check here that we only have one, and then use it's name as the name to blame if
     // anything
     // goes wrong :)
-    var embeddingProviderNames =
-        tasks.stream()
-            .map(task -> task.vectorType.getVectorizeDefinition().provider())
-            .distinct()
-            .toList();
-    if (embeddingProviderNames.size() != 1) {
-      throw new IllegalArgumentException(
-          "Must be single embedding provider name, got " + embeddingProviderNames);
+    var vectorizeIdentities =
```

Comment:
Add a comment linking issue URL and brief explanation, to indicate this is temporary limitation.